lib/good-influx-udp.js

```diff
@@ -12,6 +12,9 @@ const internals = {
 
 class GoodInfluxUdp extends GoodUdp {
     constructor(endpoint, config) {
+        if (isNaN(config.threshold) || config.threshold > 5) {
```

Comment:
Please mirror the comment from the README here:
// Limit threshold for UDP protocol to avoid transmission errors